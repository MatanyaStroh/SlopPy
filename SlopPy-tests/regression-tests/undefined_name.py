x = undefined_name
assert type(x) is NA
