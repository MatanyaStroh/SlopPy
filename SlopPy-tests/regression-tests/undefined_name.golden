{NA <type 'exceptions.NameError'> "name 'undefined_name' is not defined"
 Traceback (most recent call last):
  File "regression-tests/undefined_name.py", line 1, in <module>
    x = undefined_name
}
